### ISSUE TITLE
3D Symbology

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/styles-and-symbology#3]
We need to include 3D on the roadmap.

Suggestion to first start by testing basic Styling & Symbology with 3D content, then look into extensions for symbolizing capabilities specific to 3D.

Proposal to establish liaison with ISG DWG and CDB SWG.
Look into new 3D Tiles styling capabilities as well.

**Comment by nicolas-heigvd** [comment]
Why not directly think of a 'N-D' symbology (and how it could relate to --or extend-- the current state of SymCore)?

Or at least '4D' for features which symbology has to change with time (or any other extra dimension)?

This could be useful for Augmented/Virtual/Mixed realities.

**Comment by jerstlouis** [comment]
See

https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Jan-Erik_Vinje-From%20Points-of-interest_to_maps_of_objects.pdf

https://www.w3.org/2020/maps/supporting-material-uploads/presentations/M4W_GeoPoseUseCases_Perey_LIeberman.pdf

**Comment by chris-little** [comment]
@jerstlouis I think nD symbols need not be tied to nD data/maps. For a real example: on operational weather displays, which are 2D, lightning strikes are displayed in the correct location using an international ly agreed symbol, but it fades and disappears, so that it is 3D (2D+time). Lots of TV weather symbols are similar (raindrops or snowflakes that move)

Also, in most Augmented Reality and in many VR systems, there is a dynamic 3D objects labelled with a 2D label is some 'subtitle plane'.

So, in a conceptual model, dimensions of symbols are independent of the dimensons of the display.

**Comment by jerstlouis** [comment]
@chris-little agreed, that's what I meant by "animated symbol" in #7 (it is the symbolizer that has a temporal aspect in this case).
